addons/pylons/XEH_postInit.sqf

```diff
@@ -43,4 +43,13 @@
 
         addMissionEventHandler ["HandleDisconnect", LINKFUNC(handleDisconnect)];
     };
+
+    GVAR(searchDistanceSqr) = GVAR(searchDistance) ^ 2;
+}] call CBA_fnc_addEventHandler;
+
+["ace_settingChanged", {
```

Comment:
This EH may be needed with CBA settings but currently you have to restart the mission/ server to change `GVAR(searchDistance)` anyway.

Reply:
No harm in having it though?

Reply:
It's as harmful as any block of code that's _never_ going to be called.

Reply:
But really, it's better to not add it. The CBA settings PR should take care of this part.